examples/miral-shell/miral-xrun.sh

```diff
@@ -27,11 +32,17 @@ export GDK_BACKEND=x11
 export QT_QPA_PLATFORM=xcb
 export SDL_VIDEODRIVER=x11
 
-if [ "${x11_server}" == "Xmir" ];
+x_server_installed=$(which ${x11_server} | wc -l)
+if [ "${x_server_installed}" == "0" ]
 then
-  x_server_installed=$(apt list xmir 2>/dev/null | grep installed | wc -l)
-  if [ "${x_server_installed}" == "0" ]; then echo "Need Xmir - run \"sudo apt install xmir\""; exit 1 ;fi
+    echo "Error: Need ${x11_server}"
+    echo "On Ubuntu run \"sudo apt install xmir xwayland\""; 
+    echo "On Fedora run \"sudo dnf install xorg-x11-server-Xwayland\""; 
```

Comment:
Could compare `lsb_release -is` and provide the right command straight away?

Also could just say `Xwayland` instead of `${x11_server}`, since there's no way to reach this otherwise, and anyway you only mention xwayland packages below.

Reply:
While we *could* add that, I'm not sure that we need to.